src/Microsoft.ML.LightGbm/LightGbmCatalog.cs

```diff
@@ -67,6 +68,22 @@ public static class LightGbmExtensions
             return new LightGbmRegressionTrainer(env, options);
         }
 
+        /// <summary>
+        /// Create <see cref="LightGbmRegressionTrainer"/> from a pre-trained LightGBM model, which predicts a target using a gradient boosting decision tree regression.
+        /// </summary>
+        /// <param name="catalog">The <see cref="RegressionCatalog"/>.</param>
+        /// <param name="lightGbmModel"> A pre-trained <see cref="System.IO.Stream"/> of a LightGBM model file inferencing</param>
+        /// <param name="featureColumnName">The name of the feature column. The column data must be a known-sized vector of <see cref="System.Single"/>.</param>
+        public static LightGbmRegressionTrainer LightGbm(this RegressionCatalog.RegressionTrainers catalog,
```

Comment:
What happen if loading an LightGBM model newer than 2.3.1

Reply:
So at least "officially" we aren't going to support it. But @torronen said he would test it and let us know since the model format itself should be the same. Since we are going to be updating to the latest version though this year this is only a temporary situation.